lmdeploy/serve/async_engine.py

```diff
@@ -54,7 +55,6 @@ def safe_run(self, instance_id: int, stop: bool = False):
         self.available[instance_id] = True
 
     async def get_embeddings(self, prompt):
-        prompt = self.model.get_prompt(prompt)
```

Comment:
Not sure if we should remove it. But for open-compass evaluation, we should not preprocess the prompt.

Reply:
I think it can be removed. We don't need to decorate users prompt if they only wants to get its token embeddings